devices/tuya.js

```diff
@@ -1052,4 +1052,24 @@ module.exports = [
         toZigbee: [tz.TS0210_sensitivity],
         exposes: [e.battery(), e.vibration(), exposes.enum('sensitivity', exposes.access.STATE_SET, ['low', 'medium', 'high'])],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_la2c2uo9'}],
+        model: 'MS-105Z',
+        vendor: 'Moes',
+        description: '1 Gang 2 Way ZigBee Dimmer Switch',
+        fromZigbee: [fz.moes_105z_dimmer, fz.ignore_basic_report],
+        toZigbee: [tz.moes_105z_dimmer],
+        meta: {turnsOffAtBrightness1: true},
+        whiteLabel: [
+            {vendor: 'Tuya', model: 'MS-105Z', description: '1 Gang 2 Way ZigBee Dimmer Switch'},
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
+        },
+        exposes: [
```

Comment:
Change this to `exposes: [e.light_brightness()],`

Reply:
Changed to e.light_brightness(). 
But there is an issue with Home Assistant. Entity 'brightness' is not visible in HA, only 'light' switch and 'linkquality' sensor are visible. If I expose 'e.switch()' and 'exposes.numeric(...)' everything is ok and visible in HA.

Versions:
- Home Assistant: 2021.6.6
- z2m addon: 1.20.0-1

z2m with e.light_brightness():
<img width="679" alt="Screenshot 2021-07-03 at 12 35 28" src="https://user-images.githubusercontent.com/15796885/124350863-bed97d80-dbff-11eb-8bff-610791c7afeb.png">

HA with e.light_brightness():
<img width="330" alt="Screenshot 2021-07-03 at 12 33 28" src="https://user-images.githubusercontent.com/15796885/124350868-c4cf5e80-dbff-11eb-9a73-279841bd64f4.png">

There was with 'e.switch()' and 'exposes.numeric(...)':
<img width="336" alt="Screenshot 2021-07-03 at 12 20 08" src="https://user-images.githubusercontent.com/15796885/124350905-fa744780-dbff-11eb-9b02-8544befa7656.png">

Reply:
This is how Home Assistant works, the `state` and `brightness` can be controlled through the light entity (the one with the light bulb icon)